mir_eval/util.py

```diff
@@ -686,6 +686,40 @@ def validate_events(events, max_time=30000.):
         raise ValueError('Events should be in increasing order.')
 
 
+def validate_frequencies(frequencies, max_freq=5000., min_freq=20.):
```

Comment:
So, I'd actually prefer not to have default values for these kwargs here, because they are already defined as top-level CONSTANTS in multipitch.